# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/cherry_picks.txt
# Commit: 754436d8b3a2
# Full Hash: 754436d8b3a2023b90eb247b3a70335e74f2fdf6
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-05-09 09:14:23
# Regressor Bug: 1633628
# File Overlap Count: 3
# Description:
#   Bug 1633628 - Vender: Don't use ClearView if we previously used dual source blending on Intel gen6. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D74137
# ==============================================================================

diff -r f8fd6f0a5102 -r 754436d8b3a2 gfx/angle/cherry_picks.txt
--- a/gfx/angle/cherry_picks.txt	Fri May 08 18:39:28 2020 +0000
+++ b/gfx/angle/cherry_picks.txt	Fri May 08 19:28:53 2020 +0000
@@ -1,3 +1,25 @@
+commit 19f1bef071c32df60d9d62a7907c42270d0a3dd7
+Author: Jeff Muizelaar <jrmuizel@gmail.com>
+Date:   Fri May 8 15:08:24 2020 -0400
+
+    Handle a null blend state. (#23)
+    
+    Null is used as the default blend state so we have to check for it.
+
+commit 8df54289d717eb0624a0ee16fd681c73a9472af4
+Author: Jeff Muizelaar <jrmuizel@gmail.com>
+Date:   Wed May 6 14:04:42 2020 -0400
+
+    Don't use ClearView if we previously used dual source blending on Intel gen6. (#22)
+    
+    Doing a ClearView after a dual source blend seems to cause a TDR on
+    Intel SandyBridge. Presumeably this is because the ClearView is
+    implemented as a regular draw and the driver doesn't properly set up the
+    state.
+    
+    If we detect that this is going to happen we fall back to the manual
+    draw call path. This lets us use ClearView most of the time still.
+
 commit 3885ee272685941f6dfb6cd941d550b107f033d2
 Author: Jeff Gilbert <jgilbert@mozilla.com>
 Date:   Wed Jan 22 18:42:56 2020 -0800